src/main/java/org/folio/converter/ContentType.java

```diff
@@ -32,29 +43,29 @@ public static ContentType getByName(String name) {
     return UNKNOWN;
   }
 
-  public static ContentType resolveContentType(String type, String bLvl) {
-    switch (type) {
-      case "a":
-        return (Arrays.asList("b", "i", "s").contains(bLvl))? CONTINUING: BOOKS;
-      case "t":
+  public static ContentType resolveContentType(Leader leader) {
+    switch (leader.getTypeOfRecord()) {
+      case 'a':
+        return (Arrays.asList("b", "i", "s").contains(Character.toString(leader.getImplDefined1()[0])))? CONTINUING: BOOKS;
```

Comment:
For all condition operators please use the following for (with empty spaces):
`(x >= 0) ? x : -x;`
to follow Java Code Convention.

Reply:
Got it